src/index.js

```diff
@@ -1,14 +1,70 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+let newTask;
+const arrowBtn = document.getElementById('arrow');
+const refresh = document.getElementById('refresh');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+class Todolist {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
 
-  return element;
+  addtask(dataBase) {
+    dataBase.push(this);
+  }
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+arrowBtn.addEventListener('click', () => {
+  const taskName = document.getElementById('input').value;
+  let dataBase = JSON.parse(localStorage.getItem('baseData'));
+  if (dataBase === null) {
+    dataBase = [];
+  }
+  const index = dataBase.length + 1;
+  newTask = new Todolist(taskName, false, index);
+  newTask.addtask(dataBase);
+  localStorage.setItem('baseData', JSON.stringify(dataBase));
+  document.location.reload();
+  document.preventDefault();
+});
+
+function displaytask() {
+  const dataBase = JSON.parse(localStorage.getItem('baseData'));
+
+  const unsortedList = document.querySelector('.list');
+
+  for (let i = 0; i < dataBase.length; i += 1) {
+    const list = document.createElement('li');
+    list.id = i + 1;
+    unsortedList.appendChild(list);
+
+    const checkbox = document.createElement('input');
+    checkbox.classList.add('check');
+    checkbox.type = 'checkbox';
+    list.appendChild(checkbox);
+
+    const span = document.createElement('span');
+    span.classList.add('span');
+    span.textContent = dataBase[i].description;
+    list.appendChild(span);
+
+    const removeicon = document.createElement('button');
+    removeicon.classList.add('remove');
+    removeicon.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+    list.appendChild(removeicon);
```

Comment:
Please make sure that the indexes are updated when a task is deleted as requested by the previous reviewer.

- In the illustration below I added 3 tasks. Then I removed the first task. That means that the index of the second task should be updated to 1 since it is now the first task. And the index of the third task should become 2 since it is now the second task.

![image](https://user-images.githubusercontent.com/38088818/179347887-971ec6b8-2dfd-4ae8-b6cf-63090ef641f2.png)